src/openms/include/OpenMS/FORMAT/HANDLERS/MzMLHandler.h

```diff
@@ -5143,6 +5190,10 @@ namespace OpenMS
         if (j == 0)
         {
           os << "\t\t\t\t\t\t<cvParam cvRef=\"MS\" accession=\"MS:1000016\" name=\"scan start time\" value=\"" << spec.getRT() << "\" unitAccession=\"UO:0000010\" unitName=\"second\" unitCvRef=\"UO\" />\n";
+          if (spec.getDriftTime() > 0.0)
```

Comment:
In principle we could choose to *not* write it out here iff there is a precursor and the precursor has a set drift time. In this case the drift time is implicit and stored in the precursor attribute (which is more correct according to the standard, at least in my view).